tests/queries/0_stateless/02521_to_custom_day_of_week.reference

```diff
@@ -0,0 +1,7 @@
+1	7
+1	7
+0	6
+1	0
+2	1
+1	7
```

Comment:
Hm, should we not disallow l. 6 and l. 7 (invalid mode arguments 4 and 5)

Reply:
I saw mode in toWeek is allowed to overflow. Do you think keep the same behavior with toWeek is necessary ?

Reply:
Hm, okay. Then I would say overflowing is acceptable.

(On the other hand, overflowing is unintuitive, not documented and error-prone. Perhaps that behavior should be removed. But that should be done for all functions using mode and in a separate PR).

Reply:
I'd like to check those two modes in another PR.